### DISCUSSION TITLE
Course editing is breaking - Errors Found in linked QR codes

### DISCUSSION THREAD

**Discussion by komalm** [Sunbird-Knowlg/Community#69] [Bugs]
Hello Team,

**Issue -** Course editing is breaking - Errors Found in linked QR codes.
**Site Environments Details -** SunbirdEd portal of version 4.6

This is the scenario:
1. Create a course and publish it .
2. Edit the live course and make some changes and send it for review.

It gives an error message as below

![issueQRcode](https://user-images.githubusercontent.com/3881053/191220976-c17887a5-bb6d-432d-a851-6ab76d5cd3b4.png)

It's a client error. No network call or console error was noticed. 

Kindly help us to identify the root cause and fix this issue.

cc @baljeetsingh, @vinod303, @raghupathiguduri

**Comment by vaibhavbhuva** [comment]
@komalm Please attach the more screenshot with the details.
